tools/data-handler/src/utils/clingo-parser.ts

```diff
@@ -13,6 +13,21 @@
 import { Project } from '../containers/project.js';
 import { BaseResult, ParseResult } from '../types/queries.js';
 
+export const CLINGO_VALUE_ENCODING = 'base64url';
+
+export function encodeClingoValue(value: string) {
```

Comment:
Add comment-blocks for the new exported functions.

Reply:
Stopped using base64 encoding